news/10165.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add ``gone_in_message`` kwarg to ``pip._internal.utils.deprecation:deprecated``in order
+to allow customization or omission of the "gone_in" sentence for deprecation messages.
```

Comment:
This one is not visible to the end user, so I don’t think it should use a `feature` fragment (trivial is fine). If you *must* list this in the change log, a `process` one would be more appropriate, and the message should target downstream pip distributors specifically.

Reply:
I don't care about listing this in the changelog. Would renaming this to `10165.trivial.rst` be a solution? (I'm not so familiar with this newsfile system.)

Reply:
Yup